web/app/themes/ppj/page-footer.php

```diff
@@ -43,7 +43,9 @@
                 <div class="footer">
                     <div class="footer__container">
                         <?= $footerContent; ?>
-                        <div class="footer__logo"></div>
+                        <div class="footer__logo">
+                            <img src="/app/themes/ppj/dest/img/svg/logo.svg" alt="HM Prison and Probation Services logo"/>
```

Comment:
The text used here is "Services" – but elsewhere you've used "Service".

Is HMPPS "Services" or "Service"? This should be consistent throughout all alt text.

Reply:
done